tests/unit/core/test_tracking.js

```diff
@@ -127,3 +127,43 @@ describe('getAction', () => {
     expect(getAction('whatever')).toEqual('invalid');
   });
 });
+
+describe('Do Not Track', () => {
+  it('should respect DNT when enabled', () => {
+    expect(doNotTrackEnabled({
+      _navigator: { doNotTrack: '1' },
+      _window: {},
+    })).toBeTruthy();
```

Comment:
All the use of `toBeTruthy` / `toBeFalsy` could be `toBe(true)` and `toBe(false)`.